packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -893,4 +897,51 @@ describe("TableComponent", () => {
 			expect((<StarkTableColumnFilter[]>component.filter.columns)[0].filterValue).toBe(dummyColumnFilterValue);
 		});
 	});
+
+	describe("setStyling", () => {
+		const dummyData: Array<any> = [
```

Comment:
Same here: could you use the `[]` syntax for arrays so we keep consistency in our codebase?